### DISCUSSION TITLE
추가 기능 구현

### DISCUSSION THREAD

**Discussion by Crack-Egg** [Party-Time-Job/Party-Time-Job#100] [회의록]
## 추가 기능 구현

본인이 맡았던 페이지에서 추가로 구현하고 싶은 기능들 작성 (꼭 구현할 필요는 없음! 팀미팅 때 같이 의논해보고 구현해 보면 좋을 기능들 같이 얘기해보기!)

**Comment by Crack-Egg** [comment]
## [일반회원 페이지] - 프로필 페이지

- [x] 프로필 페이지 내 오류 발생 시 보여주는 에러 페이지 제작
- [x] 페이지 이동 시 로딩 컴포넌트 구현
- [x] 프로필 등록 창 create-portal로 외부 DOM에 생성
- [x] 연락처, 선호 지역에 올바르지 않은 값 입력 시 에러 모달 생성
- [x] 드롭다운 박스 입력 기능 추가
- [x] 드롭다운 열렸을 때 외부 클릭 이벤트로 닫는 기능 추가
- [x] 프로필 등록 미 편집 완료했을 때 토스트 기능 추가
- [x] 연락처 입력 시 자동 하이픈(`-`) 추가 기능
- [x] revalidation 기능 추가

**Comment by DanteSnow** [comment]
## [전체 페이지]

- [ ] UI 리모델링 (랜딩페이지 디자인에 맞추기)
- [ ] 랜딩 페이지 메인 이미지 변경

**Comment by yunsunglee2** [comment]
## [사장님 페이지]
- [ ] 스켈레톤 추가해보기
- [ ] 리액트 훅폼의 추가기능 구현해보기
- [ ] 폼에 시급 작성하면 자리수 세 자리 씩 끊어서 돈 단위 편하게 보이게 하기
